pages/about/Calendar.html

```diff
@@ -5,6 +5,7 @@
     <link href="./assets/favicon.ico" rel="icon" type="../../image/x-icon">
     <link href="../../semantic/out/semantic.min.css" rel="stylesheet" type="text/css">
     <link href="../../assets/css/style.css" rel="stylesheet" type="text/css">
+    <link href="../../assets/favicon.ico" rel="icon" type="assets/image/x-icon">    
```

Comment:
Maybe I missed it but where does this icon get used?

Reply:
@laurafitzgerald it is used in the tab image on the web browser.

Reply:
Ah was it just missing from this page?

Reply:
@laurafitzgerald yes.